internal/experimental/registry/client.go

```diff
@@ -277,9 +277,9 @@ func (c *Client) RemoveChart(ref *Reference) error {
 }
 
 // PrintChartTable prints a list of locally stored charts
-func (c *Client) PrintChartTable() error {
+func (c *Client) PrintChartTable(columnWidth uint) error {
```

Comment:
since this is in `experimental`, this is fine. However it does seem like an awkward API design to be passing optional fields as mandatory.

Why not add this as a field to the Client object?